build.proj

```diff
@@ -5,7 +5,7 @@
     <IsTraversalBuild>true</IsTraversalBuild>
   </PropertyGroup>
   
-  <Import Project="dir.props" />
+  <Import Project="src\dir.props" />
```

Comment:
Is there any reason not to move the PropertyGroup you added to `src\dir.props` into the root `dir.props` file?  Then you could continue importing `dir.props` from the root here, which seems a bit less convoluted and more in line with the pattern of how `dir.props` is used elsewher.